crates/nu-cli/src/commands/history/history_.rs

```diff
@@ -86,7 +86,6 @@ impl Command for History {
                         Value::record(
                             record! {
                                 fields::COMMAND_LINE => Value::string(entry.command_line, head),
-                                // TODO: This name is inconsistent with create_history_record.
```

Comment:
I think the difference between `plaintext` and `sqlite` persists that the index field is either named `index` or `idx` respectively. So I would keep the comment. (or if you are already breaking the format, do another PR to harmonize those)

At the same time the meaning of the two kinds of indexes is distinct as the one is static throug the database and the other computed on the fly for the `FileBackedHistory`

Reply:
Ah yes, so I can rename ``idx`` to ``index``
Can I do this in this PR, or do you prefer to split this up?

Reply:
I would do it in two so the commit descriptions are very explicit for the question "Why is my history broken after 0.104.0" :D

Reply:
or, I'll put the comment back in this one then